nustyle.css

```diff
@@ -376,3 +384,204 @@ ul.icons2 {
   align-self: center;
   margin-top: 15px;
 }
+
+#form-button:hover {
+  background-color: #f8b791;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+#footer-hr {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  #nav-mobile {
+    display: none;
+  }
+
+  #nav-desktop {
+    display: inline;
+    align-self: center;
+  }
+
+  #nav-desktop a {
+    padding-right: 32px;
+    margin: 0;
+    font-family: Inter;
+    font-size: 15px;
+    font-weight: 600;
+    color: #fff;
+    text-decoration: none;
+  }
+
+  header img {
+    height: 18px;
+    padding-bottom: 0;
+    padding-right: 119px;
+  }
+
+  .header-logo {
+    padding-left: 154px;
+    font-size: 20px;
+    padding-bottom: 26px;
+  }
+
+  .container {
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+    background-image: url(./assets/Illustration-header.png);
+    background-position: right;
+  }
+
+  .icons {
+    flex-direction: column;
+    padding: 279px 0 21px 36px;
+  }
+
+  .intro {
+    padding-left: 71px;
+  }
+
+  .intro p {
+    font-family: 'Poppins';
+    font-size: 16px;
+    font-weight: 400;
+    padding-right: 731px;
+    margin-bottom: 251px;
+  }
+
+  .intro h1 {
+    padding-top: 192px;
+  }
```

Comment:
- [ ] Your alignment for the intro is great @Nuga0909, Yet This is how it looks like on a `990px screen`

![your](https://user-images.githubusercontent.com/69077061/199800527-74834721-ea84-4d06-959f-7bd76a34fc8e.png)

> As you can see it's a bit pushed to the left. Fixing this makes your layout even better.